src/shared/components/plots/PlotUtils.ts

```diff
@@ -107,6 +107,57 @@ export function scatterPlotSize(
     }
 }
 
+function darkenHexChannel(c:string) {
```

Comment:
should we be using a library for this?   open for discussion

Reply:
for which part?

Reply:
all of this color manipulation stuff